tests/unit/graph_learning_test.py

```diff
@@ -98,6 +99,36 @@ class MotorSystemConfigOffObject:
     )
 
 
+@dataclass
+class ObjectEpisode:
```

Comment:
Maybe we could name this `EpisodeObservations` or `OneObjectEpisodeObservations` to be a bit more explicit that this contains the observations?